cmd/lifecycle/restorer.go

```diff
@@ -48,6 +61,10 @@ func (r *restoreCmd) Args(nargs int, args []string) error {
 		r.groupPath = cmd.DefaultGroupPath(r.platform.API(), r.layersDir)
 	}
 
+	if r.analyzedPath == cmd.PlaceholderAnalyzedPath {
```

Comment:
Should we put it under an if condition for platform API >= 0.7?

Reply:
Same answer https://github.com/buildpacks/lifecycle/pull/582#discussion_r623916968 😁